dev/mp-dev.sh

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env bash
+set -euo pipefail
+# Uncomment for debugging
+# set -x
+
+# Default environment variables
+: "${MOUNTPOINT_CSI_DEV_REGION:=eu-north-1}"
+: "${MOUNTPOINT_CSI_DEV_ECR_REPO_NAME:=mp-dev}"
+: "${MOUNTPOINT_CSI_DEV_CLUSTER_NAME:=mp-dev-cluster}"
+
+usage() {
+    echo "usage: $0 <command>"
+    echo ""
+    echo "available commands:"
+    echo "  setup                    create ECR repository, EKS cluster, and deploy everything"
+    echo "  teardown                 delete ECR repository and EKS cluster"
+    echo "  deploy-helm-chart        deploy the Helm chart from source"
```

Comment:
how about we make it explicit that we deploy from source using "ECR" or the "private ECR repo"? I think it would make it easier to see the connection with deploy-containers. We could make sure the order in which they are mentioned here is the order in which we would use them in the most common use case.